src/com/lftechnology/java/training/niraj/palindrome/LargestPalindrome.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * 
+ */
+package com.lftechnology.java.training.niraj.palindrome;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Get the largest palindrome got from product of two 3-digits number
+ * 
+ * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+ *
+ */
+public class LargestPalindrome {
+
+	private static final Logger LOGGER = Logger
+			.getLogger(LargestPalindrome.class.getName());
+
+	public static void main(String[] args) {
+		int minLimit = 100;
+		int maxLimit = 999;
+		Palindrome palindromeHelper = new Palindrome(minLimit, maxLimit);
+		int largestPalindrome = palindromeHelper.getLargestPalindrome();
+		LOGGER.log(Level.INFO,
+				"The largest palindrome for the product of two 3-digits number is "
+						+ largestPalindrome);
+
```

Comment:
Some of the log() methods can take object parameters. These object parameters are inserted into the log message, before it is being logged. The merging of object parameters into the message is only performed, if the message is not filtered out, either by a Filter, or because of too low log level. This improves performance in the cases where the message is filtered out.